<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_shop</class>
 <widget class="QDialog" name="update_shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>741</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>–ü–æ –∫–∞–∫–æ–º—É –∫—Ä–∏—Ç–µ—Ä–∏—é –≤—ã–±—Ä–∞—Ç—å –¥–∞–Ω–Ω—ã–µ –¥–ª—è –∏–∑–º–µ–Ω–µ–Ω–∏—è?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newVal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>–í–≤–µ–¥–∏—Ç–µ –Ω–æ–≤–æ–µ –∑–Ω–∞—á–µ–Ω–∏–µ, –∫–æ—Ç–æ—Ä–æ–µ –∑–∞–º–µ–Ω–∏—Ç –∑–Ω–∞—á–µ–Ω–∏–µ –≤—ã–±—Ä–∞–Ω–Ω–æ–≥–æ –ø–∞—Ä–∞–º–µ—Ç—Ä–∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ù–æ–≤–æ–µ –∑–Ω–∞—á–µ–Ω–∏–µ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>–ö–∞–∫–æ–π –ø–∞—Ä–∞–º–µ—Ç—Ä –∏–∑–º–µ–Ω–∏—Ç—å?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_10">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>–ù–∞–∑–≤–∞–Ω–∏–µ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_16">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>–ê–¥—Ä–µ—Å</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_18">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>–†–∞–±–æ—á–∏–µ —á–∞—Å—ã</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>61</y>
      <width>141</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="dfg_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ù–∞–∑–≤–∞–Ω–∏–µ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dfggfd_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ê–¥—Ä–µ—Å&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dfgfgd_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–†–∞–±–æ—á–∏–µ —á–∞—Å—ã&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="name_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>–í–≤–µ–¥–∏—Ç–µ –Ω–∞–∑–≤–∞–Ω–∏–µ –º–∞–≥–∞–∑–∏–Ω–∞</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="workinghours_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>–í–≤–µ–¥–∏—Ç–µ —Ä–∞–±–æ—á–∏–µ —á–∞—Å—ã –º–∞–≥–∞–∑–∏–Ω–∞</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_2">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>197</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>–í–≤–µ–¥–∏—Ç–µ –∞–¥—Ä–µ—Å –º–∞–≥–∞–∑–∏–Ω–∞</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton { background-color: #F5F2E7; /* –Ø—Ä–∫–æ-–∫—Ä–∞—Å–Ω—ã–π */ color: black; /* –ß–µ—Ä–Ω—ã–π —Ç–µ–∫—Å—Ç */ } QPushButton:hover { background-color: #D32F2F; /* –¢–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */ } QPushButton:pressed { background-color: #B71C1C; /* –ï—â—ë —Ç–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */ }
</string>
   </property>
   <property name="text">
    <string>–í–µ—Ä–Ω—É—Ç—å—Å—è –Ω–∞–∑–∞–¥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #F5F2E7; /* –Ø—Ä–∫–æ-–∫—Ä–∞—Å–Ω—ã–π */
    color: black;              /* –ß–µ—Ä–Ω—ã–π —Ç–µ–∫—Å—Ç */
 
}

QPushButton:hover {
    background-color: #01eb00; /* –¢–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */
}

QPushButton:pressed {
    background-color: #01a800 /* –ï—â—ë —Ç–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */
}



</string>
   </property>
   <property name="text">
    <string>–ì–æ—Ç–æ–≤–æ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;üè™ &lt;/span&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;–ò–∑–º–µ–Ω–µ–Ω–∏–µ&lt;/span&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt; –º–∞–≥–∞–∑–∏–Ω–∞ &lt;/span&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;üè™&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
